bpostshm.php

```diff
@@ -47,13 +48,27 @@ public function __construct()
 		$this->name = 'bpostshm';
 		$this->need_instance = 0;
 		$this->tab = 'shipping_logistics';
-		$this->version = '1.2';
+		$this->version = '1.21.0';
 
 		$this->displayName = $this->l('bpost Shipping Manager - bpost customers only');
 		$this->description = $this->l('IMPORTANT: bpostshm module description');
 
 		parent::__construct();
 
+		$this->hooks = array(
+			'backOfficeHeader',			// displayBackOfficeHeader
+			'beforeCarrier',			// displayBeforeCarrier
+			'extraCarrier',				// displayCarrierList
+			'paymentTop',				// displayPaymentTop
+			'processCarrier',			// actionCarrierProcess
+			'newOrder',					// actionValidateOrder
+			'postUpdateOrderStatus',	// actionOrderStatusPostUpdate
+			'updateCarrier',			// actionCarrierUpdate
+			// v1.21
+			// 'adminOrder',				// displayAdminOrder
+			'orderDetailDisplayed',		// displayOrderDetail
```

Comment:
You add a new hook with this new version. You should implement an upgrade file to register it when the module is already installed